.builds/openbsd.yml

```diff
@@ -28,11 +28,14 @@ tasks:
     cd ..
     mkdir build
     cd build
-    cmake -G Ninja ..
+    cmake -G Ninja -DMIN_LOG_LEVEL=3 ..
     cmake --build . --config Debug
     ./bin/nvim --version
 - test: |
     export LC_CTYPE=en_US.UTF-8
-    cd neovim
+    # functional tests
+    cd neovim/build
+    # cmake --build . --config Debug --target functionaltest
```

Comment:
Disabled on master until we can fix the remaining issues. In particular we need `ci/common/test.sh:check_logs()` (which checks for core dumps and prints the backtrace with gdb).  Or we could ssh into sourcehut and get the backtrace manually.